docs/rum_collection/data_collected.md

```diff
@@ -24,7 +24,9 @@ There are additional [metrics and attributes that are specific to a given event
 | View       | 30 days   | A view represents a unique screen (or screen segment) on your mobile application. A view starts and stops when the `viewDidAppear(animated:)` and `viewDidDisappear(animated:)` callbacks on the `UIViewController` class are notified. Individual `UIViewControllers` are classified as distinct views. While a user stays on a view, RUM event attributes (Errors, Resources, Actions) get attached to the view with a unique `view.id`.                           |
 | Resource   | 15 days   | A resource represents network requests to first-party hosts, APIs, and third-party providers in your mobile application. All requests generated during a user session are attached to the view with a unique `resource.id`.                                                                       |
 | Error      | 30 days   | An error represents an exception or crash emitted by the mobile application attached to the view it is generated in.                                                                                                                                                                                        |
-| Action     | 30 days   | An action represents user activity in your mobile application (for example, application launch, tap, swipe, or back). Each action is attached with a unique `action.id` attached to the view it gets generated in.                                                                                                      |
+| Action     | 30 days   | An action represents user activity in your mobile application (for example, application launch, tap, swipe, or back). Each action is attached with a unique `action.id` attached to the view it gets generated in.                                                                                                                                              |
```

Comment:
```suggestion
| Action     | 30 days   | An action represents user activity in your mobile application (for example, application launch, tap, swipe, or back). Each action is attached with a unique `action.id` to the view it gets generated in.                                                                                                                                              |
```

Reply:
I believe the current text makes the most sense although I admit it's not particular clear. 
It's trying to say that an `action.id` gets attached to an Action, and that an Action gets attached to a View.

Reply:
Thanks for clarifying, just made a suggestion which can hopefully clarify this for some customers as well. I recognize it's not part of your update, so I'll approve this request and let you decide if you want to apply the change.